main.py

```diff
@@ -0,0 +1,82 @@
+import os
+
+from netsuitesdk import NetSuiteConnection
+
+from records import TextFileEncoding, MediaType
+
+
+def connect_tba():
```

Comment:
should this file be here?

Reply:
This is spike code, more-so just to share implementation details. Not to be merged in.